app/api/helpers/scheduled_jobs.py

```diff
@@ -146,6 +146,6 @@ def expire_pending_tickets_after_three_days():
     from app import current_app as app
     with app.app_context():
         db.session.query(Order).filter(Order.status == 'pending',
-                                       (datetime.datetime.today() - Order.created_at).days > 3).\
+                                       (Order.created_at + datetime.timedelta(days=3)) > datetime.datetime.now()).\
```

Comment:
Could you explain what this does here? @prateekj117

Reply:
@prateekj117  just wanna ask, wouldn't `order-today` be actually negative? Also, please make it `>=` it will be a better choice

Reply:
@prateekj117 `(datetime.datetime.today() - Order.created_at).days > datetime.timedelta(days=3)` would've sufficed.

Reply:
@mrsaicharan1 the main problem was with the days attribute. I don't get it how this condition solves it. 
Refer [this](https://stackoverflow.com/questions/28858791/sqlalchemy-filter-by-comparing-datetime-now-and-column-default-date) for more info on this.

Reply:
Also, using .today() method will take time to be 12:00 of that particular day, we should use .now() instead.

Reply:
https://docs.python.org/3/library/datetime.html#timedelta-objects
I meant to say this 
```
datetime.datetime.today().day - Order.created_at.day > 3
```

Reply:
Also, your condition is wrong.
Suppose I placed an order yesterday. Won't it be expired?

Reply:
Also, your condition seems to be wrong.
Suppose I placed an order yesterday. Won't it be expired?

Reply:
https://docs.python.org/3/library/datetime.html#timedelta-objects
This would be a better solution
```
datetime.datetime.now().day - Order.created_at.day > 3
```

Reply:
@mrsaicharan1 I read on stackoverflow that there might be a case that python datetime isn't compatible with Sqlalchemy datetime object. So I implemented it this way which shows on stackoverflow that it works. Silly mistake on the opposite equality sign part, will correct it.

Reply:
yes @prateekj117, there might be cases , but we are comparing `days`.Difference will never change. But anyways, if you change the sign, your fix works too.